src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -902,6 +906,26 @@ private bool OnRightMouseDoubleClick()
             return false;
         }
 
+        private bool OnExtraMouseDown(MouseButtonType button)
+        {
+            if (CanExecuteMacro())
+            {
+                Macro macro = Macros.FindMacro(button, Keyboard.Alt, Keyboard.Ctrl, Keyboard.Shift);
+
+                if (macro != null && button != MouseButtonType.None)
+                {
+                    if (macro.Items != null && macro.Items is MacroObject mac)
```

Comment:
`if (macro.Items is MacroObject mac)`
is enough, implicitly check for nullptr

Reply:
fixed, thanks